### PR TITLE
Porting ekg-direct metrics to new-tracing

### PR DESCRIPTION
**Pull request by jutaro**
Direct EKG is a way to record measurments without using the tracing system. We ported it over to new tracing, partly with using standard cardano-tracer, and partly with sending it directly to the EKGBackend

### CODE REVIEW

File: trace-dispatcher/src/Cardano/Logging/DocuGenerator.hs [hunk 1]
```diff
@@ -354,21 +353,21 @@ documentMarkdown (Documented documented) tracers = do
...
+    --       l   -> fromText "> Severities: "
+    --               <> mconcat (intersperse (singleton ',')
+    --                     (map (asCode . fromString . show) l))
+    --   <>
+    --     case nub ldPrivacy of
+    --       []  -> fromText "\nPrivacy:   " <> asCode (fromString (show Public))
+    --       [p] -> fromText "\nPrivacy:   " <> asCode (fromString (show p))
+    --       l   -> fromText "\nPrivacies: "
+    --               <> mconcat (intersperse (singleton ',')
+    --                     (map (asCode . fromString . show) l))
```
Comment (deepfire) [inline trace-dispatcher/src/Cardano/Logging/DocuGenerator.hs:370]: @jutaro, do we need this commented section?

**Review by deepfire — APPROVED** [review]
@jutaro, looks good -- just one comment/question!

File: trace-dispatcher/src/Cardano/Logging/DocuGenerator.hs (hunk 1 above)
Comment (jutaro) [reply trace-dispatcher/src/Cardano/Logging/DocuGenerator.hs]: Removed, hope I can recover it one day

**Comment by jutaro** [comment]
bors merge

**Pull request merged**